lib/renderers.php

```diff
@@ -6,7 +6,7 @@
  * @param string $url The link to wrap the tag in.
  * @param string $variant_class Optional extra classes can be added for styling variants.
  */
-function render_series_ui_tag( $label, $url, $variant_class = '' ) {
+function render_ui_tag( $label, $url, $variant_class = '' ) {
   ?>
   <a href="<?php echo esc_url( $url ); ?>" class="ui-tag-block ui-tag-block--no-border <?php echo esc_attr( $variant_class ); ?>">
```

Comment:
This class `ui-tag-block--no-border` is already the `$variant_class` essentially so really should be being passed as a function parameter. Feels like it would be nice to have the function actually just take `no-border` as the function parameter and then render the class out in the function, but probably then the param would need to be an array of variant classes which would all be rendered out. I don't actually know if there are other variant classes for the ui tag at the moment anyway